templates/CodeAnalysis.yml

```diff
@@ -15,6 +15,13 @@ stages:
     pool:
       vmImage: 'ubuntu-latest'
     steps:
+    - bash: |
+        echo 'Starting docker services...'
+        docker-compose up -d
```

Comment:
Should we also have an explicit step to stop the container or just rely on Azure tearing everything done after it's finished the test run?

Reply:
Added the tear-down command to match the documentation I added